### DISCUSSION TITLE
npm install not working

### DISCUSSION THREAD

**Discussion by Akhireddy** [learndevopsonline/learndevopsonline#1811] [Interview Questions]
![103E6ED5-998B-42CB-A52A-9097D88ABB10](https://github.com/learndevopsonline/learndevopsonline/assets/33176548/ad1bd9b7-c53d-4645-ad74-3c84b1dea958)
![242F5A99-14B1-4F5F-B3C0-523F86362465](https://github.com/learndevopsonline/learndevopsonline/assets/33176548/21aaad8d-50ff-4ed9-959c-1bb274e6340f)

**Comment by chaithanya1812** [comment]
Hello, Akhireddy, catalogue is running is fine.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/14253247-0587-4f65-98d8-ef68ea5f5c24)
can you check once  frontend file   /etc/nginx/default.d/roboshop.conf  whether you add proxy rule of catalogue.?

**Reply by Akhireddy** [reply]
yes I already added catalogs ip info

**Comment by Akhireddy** [comment]
Yes i have updated catalog server ip details, [
![Uploading 7867B456-F544-4B62-A15D-85A737736353.jpeg…]()
](url)

**Comment by chaithanya1812** [comment]
can you paste frontend and mongo ip address..

**Comment by Akhireddy** [comment]
![Uploading BAE3C4A4-4737-481E-9608-E64E5BDD344F.jpeg…]()

**Comment by chaithanya1812** [comment]
can you join lab session now?

**Reply by Akhireddy** [reply]
can you share me the lab link

**Reply by chaithanya1812** [reply]
https://learndevopsonline.my.canva.site/lab-help

click on join zoom

**Comment by Akhireddy** [comment]
Can you check now, i created with raghu amis
